src/Grid/utils.jl

```diff
@@ -165,6 +165,7 @@ function _create_boundaryset(f::Function, grid::AbstractGrid, top::ExclusiveTopo
             cell_idx = ff_nh_idx[1]
             facet_nr = ff_nh_idx[2]
             cell = getcells(grid, cell_idx)
+            facet_nr > length(facets(cell)) && continue
```

Comment:
We should probably add an explanation, something like
```suggestion
            # `ff_nh::AbstractMatrix`, indexed by (cell_idx, facet_nr). For mixed grids, 
            # `facet_nr` can then become to large and should be skipped as it doesn't exist. 
            facet_nr > length(facets(cell)) && continue
```
?